mindmeld/text_preparation/tokenizers.py

```diff
@@ -304,7 +304,6 @@ def get_tokenizer_by_language(language):
         Returns:
             (Tokenizer): Tokenizer Class
         """
-        if language in SPACY_SUPPORTED_LANGUAGES:
+        if language != ENGLISH_LANGUAGE_CODE and language in SPACY_SUPPORTED_LANGUAGES:
```

Comment:
Think its better we set the default to whitespace tokenizer for all languages so we know it's backwards compatible?

Reply:
Yeah, agree.

Reply:
Sure, we can do that

Reply:
The WhiteSpaceTokenizer is now the default for all languages, the documentation has also been updated accordingly